annotation-interface/components/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+interface ITabDefinition {
```

Comment:
In TypeScript, interfaces are conventionally **not** prefixed with an "I". See [this](https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface) for reference.

Reply:
Resolved in [ad0bb11](https://github.com/hpi-dhc/PharMe/pull/347/commits/ad0bb1179be193c35c45ca361f91c501f4ce87bc).